pkg/sfu/forwarder.go

```diff
@@ -570,14 +570,14 @@ func (f *Forwarder) AllocateOptimal(availableLayers []int32, brs Bitrates, allow
 					alloc.TargetLayers.Spatial = l
 				}
 			}
-			alloc.TargetLayers.Temporal = buffer.DefaultMaxLayerTemporal
+			alloc.TargetLayers.Temporal = f.maxLayers.Temporal
```

Comment:
This change to fix client preferred fps (temporary layer), can you verify it is correct? @boks1971

Reply:
It should be okay @cnderrauber , but we probably need to allow that opportunistic higher layer. I will looking at adding it back for simulcast tracks and use proper layer for DD tracks.